src/Statistics/include/Statistics/Gaussian.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef STATISTICS_GAUSSIAN_H_
+#define STATISTICS_GAUSSIAN_H_
+
+#include <cmath>
+
+namespace orbit_statistics {
+
+// Cumulative density function of the standard (zero mean, unit variance) Gaussian distribution.
+[[nodiscard]] inline double GaussianCDF(double x) { return std::erfc(-x / std::sqrt(2)) / 2; }
```

Comment:
Style guide prefers abbreviations to be considered words for the sake of capitalization: `GaussianCdf`.
I don't know how common this abbreviation is though. If not that common, consider writing if expanded.

Reply:
> I don't know how common this abbreviation is though.
In probability and statistics -- extremely common. Also, common in packages like numpy/scipy/matlab?

Reply:
Thought so, only consider the first sentence then.